prompting/baseminer/blacklist.py

```diff
@@ -37,12 +37,12 @@ def is_prompt_in_cache(self, synapse: Prompting) -> bool:
         should_blacklist = True
     else:
         caller_hotkey = synapse.dendrite.hotkey
-        self.prompt_cache[prompt_key] = (caller_hotkey, current_block)
+        self.prompt_cache[prompt_key] = current_block
```

Comment:
Any ideas how large this prompt cache can get to? If it grows continuously, then this can be a problem for servers that run a long time

Reply:
It is cleaned out every 50 blocks by default: https://github.com/opentensor/text-prompting/blob/remove-promptcache-check/prompting/baseminer/config.py#L146